README.md

```diff
@@ -5,3 +5,40 @@ Notepad++ supports around 80 programming languages for syntax highlighting & fol
 To learn all about User Defined Languages:
 
 https://npp-user-manual.org/docs/user-defined-language-system/
+
+## Using one of these User Defined Languages
+
+For now, you have to manually install a new User Defined Language (UDL).
+
+1. Download the XML file(s) from the [`UDLs`](./UDLs) folder of [this Collection](https://github.com/notepad-plus-plus/userDefinedLanguages).
```

Comment:
I would underline "it's recommended" since it's a cleaner way the manage UDLs.

Reply:
I rephrased it to make the preferred method more obvious.  Thanks